modules/core/src/config.ts

```diff
@@ -22,296 +23,52 @@ export interface Tokens {
   };
 }
 
+// Get the list of ERC-20 tokens from statics and format it properly
+const formattedErc20Tokens = coins.filter((coin: BaseCoin) => {
+  return coin instanceof Erc20Coin;
+}).map((token: Erc20Coin): TokenConfig => {
+  return {
+    type: token.name,
+    coin: token.network.type === NetworkType.MAINNET ? 'eth' : 'teth',
+    network: token.network.type === NetworkType.MAINNET ? 'Mainnet' : 'Testnet',
+    name: token.fullName,
+    tokenContractAddress: token.contractAddress.toString(),
```

Comment:
Isn't it already a string? Should lowercase it though, as it would break things here if any tokens were accidentally not lowercased in statics

Reply:
I was getting weird errors, because we use a type: ```ContractAddress``` (even though it extends string)

Reply:
Ah ok -- still lowercase it